packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts

```diff
@@ -1,12 +1,36 @@
-import { defineComponent } from 'vue'
-import ColorCSSVariablesUpdater from './ColorCSSVariablesUpdater.demo.vue'
+import { VaChip, VaButton } from '../../components'
+import { useColors } from '../../composables'
 
 export default {
-  title: 'ColorCSSVariablesUpdater',
-  component: ColorCSSVariablesUpdater,
+  title: 'services/ColorCSSVariablesUpdater',
 }
 
-export const Default = defineComponent({
-  components: { ColorCSSVariablesUpdater },
-  template: '<ColorCSSVariablesUpdater/>',
+export const Default = () => ({
+  components: { VaChip, VaButton },
+  setup () {
+    const colors = useColors()
+    const change = (color) => {
+      colors.setColors({ danger: color })
+    }
+
+    return {
+      change,
+    }
+  },
+  template: `
+    <div class="mb-6">
+      <a class="var(--va-danger)">
+        This is title danger color
+      </a>
```

Comment:
Hi, @m0ksem, I have changed code according to your comments. I have take another look, in storybook this anchor doesn't show the danger color. I Have no idea why... :(

Reply:
It is a stupid mistake, I reflect on it this morning, turns out I am using class but not style.